### ISSUE TITLE
No OSD on Cetus after FC Flash

### ISSUE THREAD

**Issue by a-di-ez** [BETAFPV/BETAFPV_Configurator#10]
Hello,

After flash firmware on Cetus (Lite 1S Pro V1.1)  I don't see any information (about fly time, mode, battery voltage) on the screen. There is only a raw view from the camera and word DISARM. Could you help me and tell me what I should do to enable this information - I don't see this option in BETAFPV Configurator (v1.1.0) Windows version.

Best regards,
Vadim Osochenko

**Comment by m4rc1no5** [comment]
Hi Vadim,

did you try the solution from the task: https://github.com/BETAFPV/BETAFPV_Configurator/issues/7 ?

**Comment by a-di-ez** [comment]
Hi, @m4rc1no5 
I don't have Cetus Pro, need OSD firmware for Cetus

**Comment by a-di-ez** [comment]
@HWT662988 any progress?

**Comment by HWT662988** [comment]
hi,a-di-ez
I'm so sorry to see your issue so late, I need to ask which firmware is suitable, I'll tell you how to do in a minute

**Comment by a-di-ez** [comment]
cetus 1.0.1 from december 2021

**Comment by HWT662988** [comment]
hi,a-di-ez
The **Lite 1-2S Pro** is the  early alias of **Cetus Pro**, I  need to know whether your FC's motor is brushed(cetus) or brushless(cetus pro)

**Comment by a-di-ez** [comment]
this is strange, because the board says: **Lite 1S Pro V1.1** 
FC's motor is brushed(cetus)

**Comment by a-di-ez** [comment]
I can send photo of FC board

**Comment by a-di-ez** [comment]
![image](https://user-images.githubusercontent.com/10465719/150093447-b4207d4e-fc38-4eeb-b1ea-2fa6158f630d.png)

**Comment by HWT662988** [comment]
OK,Is is the early version ,and the  baudrate between OSD and FC has been updated  in later versions,
here is early firmware the you can flash and test if it is available. 
[Cetus_brush_2021-08-28.zip](https://github.com/BETAFPV/BETAFPV_Configurator/files/7895490/Cetus_brush_2021-08-28.zip)

**Comment by a-di-ez** [comment]
@HWT662988 OSD is back, but now there are even more problems:
1. I can't connect the drone to configurator for change rates - Need to update the firmware
2. After turning on the remote control, the drone starts flashing green and does not respond to commands

**Comment by HWT662988** [comment]
about the first problem I will tell you how to do later.
and about the second problem: flashing green indicates your FC is in **binding** mode , you just need press the **bind** button  on you LiteRadio to binding it.

**Comment by a-di-ez** [comment]
Can I revert to later firmware version now until you resolve my problem?

**Comment by a-di-ez** [comment]
I think this is not necessary, since my old rates were preserved after the firmware upgade, is this possible?

**Comment by HWT662988** [comment]
yes , upgrade update do not change the rates .

**Comment by a-di-ez** [comment]
great, wait on your solution!)

**Comment by HWT662988** [comment]
1.you need to erase the OSD 's firmeware by the old Cetus-configurator.
[Cetus-configure.zip](https://github.com/BETAFPV/BETAFPV_Configurator/files/7895858/Cetus-configure.zip)
![image](https://user-images.githubusercontent.com/45007731/150105755-7be10ed7-49d5-4764-869e-1fa5e09c32da.png)

2.open the BETAFPV Configurator ,flash the new OSD firmware firstly.
[LiteSilverOSD_2021-10-26.zip](https://github.com/BETAFPV/BETAFPV_Configurator/files/7895976/LiteSilverOSD_2021-10-26.zip)

3.and then you can upgrade the new FC firmware .

**Comment by a-di-ez** [comment]
This checkboxes missed
![image](https://user-images.githubusercontent.com/10465719/150107877-5ba6cdb3-2a05-4f97-994c-bc810439aa38.png)

**Comment by HWT662988** [comment]
check if here:
![image](https://user-images.githubusercontent.com/45007731/150108871-679ed32a-4f79-43b6-a6ff-cd7b993698c5.png)

**Comment by HWT662988** [comment]
you just need to use the Cetus-Configurator to erase the OSD's firmware . and please do not use it to upgrade the firmware.

for upgrade the firmware you can use the BETAFPV Configurator and It can avoid some flashing mistakes on chip.

**Comment by a-di-ez** [comment]
after click on Erase OSD btn - I don't have any results or messages
Need erase in White LED color mode?

**Comment by HWT662988** [comment]
don't  need erase in White LED color mode, It will not return message. you just need click on Erase OSD btn,and come back to BETAFPV Configurator to flash the new OSD firmware

**Comment by tano68** [comment]
hi, your answers in that thread is very helpful, so i would ask you something. I'looking for a working sensor firmware for the cetus pro. The latest version available im the github isn't recognized by Configurator (like the one for OSD). Do you have a link for it ?
Thanks for your support.

**Comment by a-di-ez** [comment]
@tano68 create new issue please, your problem is not related to Cetus (Not PRO)

**Comment by a-di-ez** [comment]
@HWT662988 your solution doesn't work, any idea?

**Comment by HWT662988** [comment]
hi, a-di-ez
please use this new configurator to erase the OSD 's firmeware.
[configure_V3.zip](https://github.com/BETAFPV/BETAFPV_Configurator/files/7901487/configure_V3.zip)

![150105755-7be10ed7-49d5-4764-869e-1fa5e09c32da](https://user-images.githubusercontent.com/45007731/150245048-03b161ab-1ad6-4e79-a8f8-766597a21083.png).


please do **not**  erase the other firmeware (Sensor and FC).

**Comment by HWT662988** [comment]
@tano68  hi,tano68
you can create a new issue for this problem.

For sensor firmware, It is not recommended to be updated, the factory firmware of sensor is the most suitable.
and the latest version you saw just for compatibility with the changes of chip.

**Comment by a-di-ez** [comment]
@HWT662988 erased successfully, load Lite OSD successfully too, but OSD does't work(

**Comment by HWT662988** [comment]
please load and  flash the OSD firmware again. use the BETAFPV Configurator. The osd firmware is named "LiteSilverOSD_2021-10-26" which and be download above. flashing OSD need be  in the White LED color mode.

**Comment by a-di-ez** [comment]
It's strange that you tell me to flash in White LED color mode, because in another LED mode the firmware is impossible
But I again flashed OSD in White LED color mode and of course no change in the end

**Comment by a-di-ez** [comment]
I found a solution, need to use Configurator V3 with this firmware Cetus_brush_2021-08-28.zip
In result I can change Rates and see OSD
New Betafpv_Configurator v1.1.0 doesn't work with Cetus (Lite 1S Pro V1.1)

**Comment by a-di-ez** [comment]
I found a solution, need to use Configurator V3 with this firmware Cetus_brush_2021-08-28.zip
In result I can change Rates and see OSD
New Betafpv_Configurator v1.1.0 doesn't work with Cetus (Lite 1S Pro V1.1)
